qa/TL1_paddle_tsm/test.sh

```diff
@@ -17,7 +17,7 @@ do_once() {
 
 test_body() {
   out=`python infer.py -k 1 -s 15 demo`
-  if echo $out | grep -E "(paragliding|sticking_tongue_out)"; then
+  if echo $out | grep -E "(waxing_legs|paragliding)"; then
```

Comment:
that's indeed a drastic change between the versions:)

Reply:
In CI it returns `paragliding`, locally I get `waxing_legs`. Visually examining the clip doesn't give me much clue which one is more correct or both are equally not correct (and I get different result buy just updating PaddlePaddle and changing the model loading from `paddle.fluid.io.load_persistables(exe, '', prog, filename=weight_path)` to `paddle.distributed.io.load_persistables(exe, '', prog, filename=weight_path)`) :
![image](https://github.com/NVIDIA/DALI/assets/39967756/43744403-e512-48b8-b1bc-e669ef3c302e)
![image](https://github.com/NVIDIA/DALI/assets/39967756/f0702fb7-2dd6-4349-821f-7efc964dd027)